src/views/List.jsx

```diff
@@ -1,23 +1,41 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	/* TO DO: Make separate resuable input component with a filter feature*/
+
+	const filterList = data.filter((item) => {
```

Comment:
One minor potential improvement could be to filter the list for _uniqueness_ in addition to letter case. That way the user isn't show multiple instances of the same item. A similar check for uniqueness could also be implemented at the point where the user attempts to add a new item.

Reply:
This is a good callout @jongranados! I believe a future issue addresses item uniqueness, so I'm not sure it is completely necessary here.

Reply:
@djtaylor8 is right! In a couple weeks, you're going to make it so that a user cannot enter save duplicate item in the first place. This prevention in the API layer means we won't have to think about it in the rendering layer. It's great that you're thinking about this case, though, @jongranados 💜